recipes/hail/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hail" %}
+{% set version = "0.2.34" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 9c3a4e5f81ad23b569c1d6dc80bfb070
```

Comment:
Please use the `sha256`. It should be:

9ca5a9018208b048588c2cad6312aeb86bb123d7ef76919977fd47c02e7d5587